api/src/controllers/authController.ts

```diff
@@ -28,34 +27,37 @@ export default async (fastify: FastifyInstance) => {
     url: '/register',
     schema: fastify.getSchema(ESchemasIds.POSTRegister),
     handler: async function (
-      request: Request<{ Body: { pseudo: string, mail: string, password: string } }>,
+      request: Request<{
+        Body: { pseudo: string; mail: string; password: string };
+      }>,
       reply: Reply
     ) {
       const { _errorService, _authService } = this;
       const { body } = request;
 
       // Duplicate check
-      const user = await _authService.getUserByPseudoOrMail(body.pseudo, body.mail);
+      const user = await _authService.getUserByPseudoOrMail(
+        body.pseudo,
+        body.mail
+      );
       if (user && body.mail === user.mail)
         _errorService.send(EErrorMessages.DUPLICATE_MAIL, 409);
       if (user && body.pseudo === user.pseudo)
         _errorService.send(EErrorMessages.DUPLICATE_PSEUDO, 409);
-      
+
       // Test and Hash password
       if (!body.password.match(process.env.PASS_REGEXP))
         _errorService.send(EErrorMessages.INVALID_PASSWORD_FORMAT, 422);
-      
+
       body.password = await _authService.hashString(body.password);
 
       // Create user
       await _authService.createUser(body);
 
-      reply
-        .code(200)
-        .send({ message: EResponseMessages.CREATE_USER_SUCCESS });
+      reply.code(200).send({ message: EResponseMessages.CREATE_USER_SUCCESS });
```

Comment:
```suggestion
      reply
        .code(200)
        .send({ message: EResponseMessages.CREATE_USER_SUCCESS });
```

Try to keep the code structure clean, it's like that everywhere else in the API there is no reason to change that only here.